src/diffusers/pipelines/kandinsky/pipeline_kandinsky.py

```diff
@@ -371,4 +398,5 @@ def __call__(
 
             _, latents = latents.chunk(2)
 
-        return latents
+        images = self.get_image(latents)
```

Comment:
let's do post-processing similar to what's done here https://github.com/huggingface/diffusers/blob/kandinsky/src/diffusers/pipelines/kandinsky/pipeline_kandinsky_inpaint.py#L452